src/lib/FreeC/Pretty.hs

```diff
@@ -80,23 +74,22 @@ instance Pretty a => Pretty (TrailingLine a) where
 -------------------------------------------------------------------------------
 -- Rendering                                                                 --
 -------------------------------------------------------------------------------
-
 -- | Pretty prints a value with a maximum line length of @120@ characters of
 --   which @80@ are allowed to be non-indentation characters.
 renderPretty' :: Pretty a => a -> SimpleDoc
 renderPretty' = renderPretty ribbonFrac maxLineWidth . pretty
  where
-  ribbonWidth, maxLineWidth :: Int
-  ribbonWidth  = 80
-  maxLineWidth = 120
+   ribbonWidth, maxLineWidth :: Int
+   ribbonWidth = 80
+
```

Comment:
I think the empty line looks rather odd in this case, where the local definitions are just constants (and even share their type signature).

Reply:
There is an option to remove blank lines in `where`-blocks, but that would also remove the blank lines in `where`-blocks that contain more complex functions. I suggest to move the type signature instead.

```haskell
 where
   ribbonWidth :: Int
   ribbonWidth = 80

   maxLineWidth :: Int
   maxLineWidth = 120
```